src/resources/packages/wizard/components/tabs/organizer/tab.tsx

```diff
@@ -150,11 +150,15 @@ const OrganizerContent = ({moveToNextTab, skipToNextTab}) => {
 
 		let isValid = false;
 
-		try {
-			const url = new URL(website);
-			isValid = url.protocol === 'http:' || url.protocol === 'https:';
-		} catch (e) {
-			isValid = false
+		if ( website === '' ) {
+			isValid = true;
+		} else {
+			try {
+				const url = new URL(website);
+				isValid = url.protocol === 'http:' || url.protocol === 'https:' || url == '';
+			} catch (e) {
+				isValid = false
+			}
 		}
```

Comment:
No need for the `else`:
```suggestion
		if ( website === '' ) {
			return true;
		}
		
		try {
			const url = new URL(website);
			isValid = url.protocol === 'http:' || url.protocol === 'https:' || url == '';
		} catch (e) {
			isValid = false
		}
```

Reply:
🤦